index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  </head>
+
+  <body>
+    <h1>Hello World!we are software developers</h1>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>Welcome</h1>
+    </section>
+    <section id="projects">
+      <div class="project-tile"> </div>
+      <a href="/project-1">Project 1</a>
+    </section>
+    <nav id="navbar">
+      <a href="#welcome-section">Welcome Section</a>
+    </nav>
+    <a id="profile-link" target="_blank" href="https://www.github.com/munsa1" rel="noopener">GitHub</a>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
remove unnecessary comments

Reply:
As a code reviewer, I think it is best you politely suggest changes to the author of a project. If you politely suggest changes, you present your ideas less authoritatively and give the author the chance to be at ease when communicating with you. For example, your comment about removing commented code could be presented like this:

> I want to kindly suggest that you all unnecessary comments from all files because it makes the code look very clean.